package.json

```
@@ -54,6 +53,9 @@
     "ts-jest": "^26.1.3",
     "typescript": "^4.0.2"
   },
+  "peerDependencies": {
+    "@octokit/core": "^3.0.0"
```

please change to 

```suggestion
    "@octokit/core": ">=3"
```

the plugin will most likely continue to work with v4 and beyond. And if not, we will adopt the peer dependency range accordingly